src/components/header/header.css

```
@@ -0,0 +1,34 @@
+.header {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding: 1vw;
```

что-то странное, ломающее верстку
Почему просто не сделать
```suggestion
    padding: 15px 0;
```

или хотяб добавить box-sizing